packages/world/src/resourceTypes.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+bytes2 constant RESOURCE_NAMESPACE = "ns";
+bytes2 constant RESOURCE_TABLE = "tb";
+bytes2 constant RESOURCE_TABLE_OFFCHAIN = "tf";
```

Comment:
why not `to` or `ot` 👀

Reply:
considered it but "offchain" and "onchain" are both "o"

also `tb` and `to` look too similar and wanted to visually differentiate

🤷

Reply:
fair, `f` is the only diff between `onchain` and `offchain` 🙈

Reply:
@dk1a in case you weren't following our chat, we probably want to define the table ID prefix separately and inside store, and the other resource prefixes here in world